fastlane/lanes/release_management_in_ci.rb

```diff
@@ -56,7 +58,7 @@
     buildkite_trigger_build(
       buildkite_organization: BUILDKITE_ORGANIZATION,
       buildkite_pipeline: BUILDKITE_PIPELINE,
-      branch: "#{editorial_branch}",
+      branch: editorial_branch.to_s,
```

Comment:
I suppose RuboCop did this automatically.

I tried it locally, and we should be able to go with a straight access:

```suggestion
      branch: editorial_branch,
```

Notice that `editorial_branch.is_a? String` prints `true` in my local test:

<img width="1280" alt="image" src="https://github.com/wordpress-mobile/WordPress-Android/assets/1218433/91cb523f-e593-4c8a-9ad1-69bcb3399c08">